index.js

```diff
@@ -1,4 +1,4 @@
 'use strict';
 
-// https://regex101.com/r/SQrOlx/12
-module.exports = () => /(?:\w[\w-.]+\/\w[\w-.]+|\B)#[1-9]\d*\b/g;
+// https://regex101.com/r/SQrOlx/13
+module.exports = () => /(?:(?<=[\s<[(])\w[\w-.]+\/\w[\w-.]+|\B)#[1-9]\d*\b/g;
```

Comment:
`\s<[(` is too specific to the fixture. Yes, it works, but there are many other real-world scenarios it misses by being so strict. The regex in #5 instead just says that anything is allowed before the issue except for a valid issue character.